scripts/sct_resample.py

```diff
@@ -1,31 +1,24 @@
-#!/usr/bin/env python
 #########################################################################################
 #
 # Resample data.
 #
 # ---------------------------------------------------------------------------------------
 # Copyright (c) 2014 Polytechnique Montreal <www.neuro.polymtl.ca>
 # Authors: Julien Cohen-Adad, Sara Dupont
-# Modified: 2015-09-08
+# Modified: 2017-11-22 (moved the code to spinalcordtoolbox module)
 #
+# 
 # About the license: see the file LICENSE.TXT
 #########################################################################################
 
-# TODO: adjust qform of output data to account for pixel size
-# TODO: test if crashes with 2d or 4d data
-# TODO: raise exception if input size is not numerical
-
 import sys
-# import os
-# import getopt
+
 import commands
```

Comment:
Unused

Reply:
which one ? the `sys` and `commands` are being used